CRM/Core/xml/Menu/Admin.xml

```diff
@@ -101,6 +101,12 @@
      <page_callback>CRM_UF_Form_Group</page_callback>
      <weight>25</weight>
   </item>
+  <item>
+     <path>civicrm/admin/uf/group/copy</path>
+     <title>Profile Copy</title>
+     <page_callback>CRM_UF_Page_Group</page_callback>
```

Comment:
I think it would be better to refer directly to the form class so the page class doesn't need to be overloaded anymore.
```suggestion
     <page_callback> CRM_UF_Form_Group </page_callback>
```